OpenRA.Mods.Common/Orders/PlaceBuildingOrderGenerator.cs

```diff
@@ -306,15 +306,26 @@ IEnumerable<Order> ClearBlockersOrders(World world, CPos topLeft)
 
 			var blockers = allTiles.SelectMany(world.ActorMap.GetActorsAt)
 				.Where(a => a.Owner == queue.Actor.Owner && a.IsIdle)
-				.Select(a => new TraitPair<Mobile>(a, a.TraitOrDefault<Mobile>()));
+				.Select(a => new TraitPair<IMove>(a, a.TraitOrDefault<IMove>()));
 
 			foreach (var blocker in blockers.Where(x => x.Trait != null))
```

Comment:
Also out of this scope, but we could move this Where up to the rest of the query.